python/README.md

```diff
@@ -1,59 +1,83 @@
 # Welcome to the CUDA Quantum Python API
 
-## Installing
+## Installing CUDA Quantum
 
-Programmers may use pip to handle the installation of the CUDA Quantum Python
-API and any of the dependencies needed to do so. This may be done either from
-source or through our binary wheel distribution. We currently support the
-following operating systems: Linux.
-
-### Pip Wheels
-
-Programmers may install CUDA Quantum via the command line:
+To pull down the CUDA Quantum Python wheel distribution, simply run:
 
 ```console
 pip install cuda-quantum
 ```
 
-### Pip install from source
+At this time, wheels are distributed for Linux operating systems only.
 
-You may also install from source as follows:
+To build a source distribution of the Python API through pip:
 
 ```console
 git clone https://github.com/NVIDIA/cuda-quantum.git
 cd cuda-quantum
-pip install . --user
+pip install . 
 ```
 
-This will install any dependencies and build the necessary pieces of the
-CUDA Quantum repository through cmake. It will then install the
-`cuda-quantum` python package through your pip package manager.
-
-### Build entirely from source
-
-If you would like to avoid the use of pip, or you want to build the entire C++ API
-with the Python API, follow the [installation instructions][official_install]. You
-may then export your python path to point to the installation directory:
+If you would like to build the entire C++ and Python API's from source,
+follow the [installation instructions][official_install]. You
+may then export your python path to the installation directory:
 
 ```console
 export PYTHONPATH=$PYTHONPATH:/path/to/cudaq/install
 ```
 
 [official_install]: https://nvidia.github.io/cuda-quantum/latest/install.html
 
-## Getting started
+## Running CUDA Quantum
 
-You should now be able to import CUDA Quantum and start building quantum programs
-in Python!
+You should now be able to import CUDA Quantum and start building quantum programs in
+Python!
 
 ```console
 import cudaq
 
 kernel = cudaq.make_kernel()
+qubit = kernel.qalloc()
+kernel.x(qubit)
+kernel.mz()
+
+result = cudaq.sample(kernel)
 ```
 
-To see more examples, [go to][python_examples], or check out the
+## Documentation 
+
+To see more examples, go to [python examples][python_examples], or check out the
 [Python API reference][python_api_reference].
 
 [python_examples]: https://nvidia.github.io/cuda-quantum/latest/using/python.html
 [python_api_reference]: https://nvidia.github.io/cuda-quantum/latest/api/languages/python_api.html
+
+## Contributing
+
+There are many ways in which you can get involved with CUDA Quantum. If you are
+interested in developing quantum applications with CUDA Quantum, this repository
```

Comment:
all the "this repository" in the readme needs to be replaced. The license section in particular doesn't seem applicable. Remove the section about the CLA bot. You can state the source code is available on GitHub and licensed under Apache 2.0. Let's maybe mirror the phrasing on NGC and also state 

"CUDA Quantum is an open source project. The source code is available [on GitHub](https://github.com/NVIDIA/cuda-quantum) and licensed under [Apache 2.0](https://github.com/NVIDIA/cuda-quantum/blob/main/LICENSE) license. By downloading NVIDIA CUDA Quantum, you accept the terms and conditions of the NVIDIA End User License Agreement. "

Reply:
Ok I've updated everything but you should definitely give them another look over when you can